components/client/src/main/java/com/hotels/styx/client/connectionpool/StatsReportingConnectionPool.java

```diff
@@ -98,15 +103,12 @@ private void registerMetrics() {
         }
     }
 
-    private static void registerMetrics(ConnectionPool hostConnectionPool, MetricRegistry scopedRegistry) {
-        ConnectionPool.Stats stats = hostConnectionPool.stats();
-        scopedRegistry.register("busy-connections", (Gauge<Integer>) stats::busyConnectionCount);
-        scopedRegistry.register("pending-connections", (Gauge<Integer>) stats::pendingConnectionCount);
-        scopedRegistry.register("available-connections", (Gauge<Integer>) stats::availableConnectionCount);
-        scopedRegistry.register("connection-attempts", (Gauge<Integer>) () -> (int) stats.connectionAttempts());
-        scopedRegistry.register("connection-failures", (Gauge<Integer>) () -> (int) stats.connectionFailures());
-        scopedRegistry.register("connections-closed", (Gauge<Integer>) () -> (int) stats.closedConnections());
-        scopedRegistry.register("connections-terminated", (Gauge<Integer>) () -> (int) stats.terminatedConnections());
+    private void removeMetrics() {
```

Comment:
I would be happier if we didn't have to repeat the list of metrics here.  Accepting for now...

Reply:
I agree.